django/common/scripts/merging/merge_instant.py

```diff
@@ -0,0 +1,7 @@
+from common.scripts.cleaning import clean_instant
+
+
+def merge_instant(*args):
+    """Merging script with a instant concatenation"""
+    values = [v for v in args if v is not None]
+    return clean_instant("T".join(values))
```

Comment:
only works when merging 2 values right ? it won't be valid if there are more than 2

Reply:
Yup! I took the same pattern as merge_datetime: https://github.com/arkhn/fhir-river/blob/main/django/common/scripts/merging/merge_datetime.py

Generally, we want to merge a date with a time, I never saw a case where we want to merge more than two values

Reply:
then shouldn't you raise an error when there are more than two ?

Reply:
@simonvadee should I use `logging.warning` or `raise error` ?